{
    "version": "1.0.0",
    "plays": [],
    "stdout": [
        "\u001b[0;34mansible-playbook [core 2.12.1]\u001b[0m",
        "\u001b[0;34m  config file = /etc/ansible/ansible.cfg\u001b[0m",
        "\u001b[0;34m  configured module search path = ['/home/runner/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']\u001b[0m",
        "\u001b[0;34m  ansible python module location = /usr/lib/python3.8/site-packages/ansible\u001b[0m",
        "\u001b[0;34m  ansible collection location = /home/runner/.ansible/collections:/usr/share/ansible/collections\u001b[0m",
        "\u001b[0;34m  executable location = /usr/bin/ansible-playbook\u001b[0m",
        "\u001b[0;34m  python version = 3.8.8 (default, Aug 11 2021, 06:52:42) [GCC 8.5.0 20210514 (Red Hat 8.5.0-3)]\u001b[0m",
        "\u001b[0;34m  jinja version = 2.10.3\u001b[0m",
        "\u001b[0;34m  libyaml = True\u001b[0m",
        "\u001b[0;34mUsing /etc/ansible/ansible.cfg as config file\u001b[0m",
        "\u001b[0;34mhost_list declined parsing /etc/ansible/hosts as it did not pass its verify_file() method\u001b[0m",
        "\u001b[0;34mscript declined parsing /etc/ansible/hosts as it did not pass its verify_file() method\u001b[0m",
        "\u001b[0;34mauto declined parsing /etc/ansible/hosts as it did not pass its verify_file() method\u001b[0m",
        "\u001b[0;34mParsed /etc/ansible/hosts inventory source with ini plugin\u001b[0m",
        "\u001b[1;35m[WARNING]: provided hosts list is empty, only localhost is available. Note that\u001b[0m",
        "\u001b[1;35mthe implicit localhost does not match 'all'\u001b[0m",
        "\u001b[0;34mredirecting (type: modules) ansible.builtin.snow_record_find to servicenow.servicenow.snow_record_find\u001b[0m",
        "\u001b[0;31mERROR! couldn't resolve module/action 'snow_record_find'. This often indicates a misspelling, missing collection, or incorrect module path.\u001b[0m",
        "\u001b[0;31m\u001b[0m",
        "\u001b[0;31mThe error appears to be in '/home/truch/ansible-examples/ServiceNowQuery/site.yml': line 6, column 5, but may\u001b[0m",
        "\u001b[0;31mbe elsewhere in the file depending on the exact syntax problem.\u001b[0m",
        "\u001b[0;31m\u001b[0m",
        "\u001b[0;31mThe offending line appears to be:\u001b[0m",
        "\u001b[0;31m\u001b[0m",
        "\u001b[0;31m  tasks:\u001b[0m",
        "\u001b[0;31m  - name: Find open standard changes with my template\u001b[0m",
        "\u001b[0;31m    ^ here\u001b[0m"
    ],
    "status": "failed",
    "status_color": 9
}